packages/web-components/src/components/va-accordion/va-accordion-item.tsx

```diff
@@ -35,28 +40,41 @@ export class VaAccordionItem {
    */
   @Prop() level: number = 2;
 
+  @State() slotHeader: string = null;
+
+  @State() slotTag: string = null;
+
   private toggleOpen(e: MouseEvent): void {
     this.accordionItemToggled.emit(e);
   }
 
+  private populateStateValues() {
```

Comment:
Can we also have a comment saying why we're using a `slotchange` handler?

Reply:
Updated